examples/simple_system/README.md

```diff
@@ -6,6 +6,7 @@ run stand-alone binaries. It contains:
 * An Ibex Core
 * A single memory for instructions and data
 * A basic peripheral to write ASCII output to a file and halt simulation from
+* A basic timer peripheral capable of generating interrupts
   software
```

Comment:
worth adding something that says this follows the behaviour specified in the 'Machine Timer Registers' in the RISCV privileged spec

Reply:
Done